frontend_lib/src/component/PopinFixed/PopinFixedContent.jsx

```diff
@@ -17,41 +27,169 @@ class PopinFixedContent extends React.Component {
+              loggedUser={props.loggedUser}
+              content={props.content}
+              favoriteState={props.favoriteState}
+              onClickAddToFavoriteList={props.onClickAddToFavoriteList}
+              onClickRemoveFromFavoriteList={props.onClickRemoveFromFavoriteList}
+            />
+            <div className={classnames('wsContentGeneric__content__left__top', `${props.customClass}__content__left__top`)}>
+              {props.breadcrumbsList.length > 0 && (
+                <Breadcrumbs
+                  root={{
+                    link: PAGE.HOME,
+                    label: '',
+                    icon: 'fas fa-home',
+                    type: BREADCRUMBS_TYPE.CORE,
+                    isALink: true
+                  }}
+                  breadcrumbsList={props.breadcrumbsList}
+                />
+              )}
+
+              {!!props.lastVersion &&
+                (props.appMode === APP_FEATURE_MODE.VIEW || props.appMode === APP_FEATURE_MODE.REVISION) &&
+                (
+                  <div
+                    className={classnames(
+                      'wsContentGeneric__content__left__top__version',
+                      `${props.customClass}__content__left__top__version`
+                    )}
+                  >
+                    {props.t(
+                      'Version #{{versionNumber}}', {
+                        versionNumber: props.appMode === APP_FEATURE_MODE.VIEW && !props.isRefreshNeeded
+                          ? props.lastVersion
+                          : props.content.number
+                      }
+                    )}
+                    {(props.appMode === APP_FEATURE_MODE.REVISION || props.isRefreshNeeded) && (
+                      <div
+                        className={classnames(
+                          'wsContentGeneric__content__left__top__lastversion',
+                          `${props.customClass}__content__left__top__lastversion`
+                        )}
+                      >
+                        ({props.t('latest version: {{versionNumber}}', { versionNumber: props.lastVersion })})
+                      </div>
+                    )}
+                    &nbsp;-&nbsp;
+                  </div>
+                )}
+
+              {props.availableStatuses && props.loggedUser.userRoleIdInWorkspace >= ROLE.contributor.id && (
+                <SelectStatus
+                  selectedStatus={props.availableStatuses.find(s => s.slug === props.content.status)}
+                  availableStatus={props.availableStatuses}
+                  onChangeStatus={props.onChangeStatus}
+                  disabled={props.appMode === APP_FEATURE_MODE.REVISION || props.content.is_archived || props.content.is_deleted}
+                />
+              )}
+            </div>
+
             {this.props.children[0]}
           </div>
 
-          {React.cloneElement(this.props.children[1], {
+          {React.cloneElement(props.children[1], {
```

Comment:
A comment here would help understand the usage of `cloneElement` which is not frequent.

Reply:
This code was not done in this PR, I will add this in an issue.

Reply:
https://github.com/tracim/tracim/issues/4750